lib/client.js

```diff
@@ -26,6 +27,9 @@ var backoff = require('backoff-func');
 var _resources = require('./resources.js');
 var _utils = require('./utils.js');
 
+var hostIsNotReachableErrorCodes =
+    ['ETIMEDOUT', 'ENOTFOUND', 'ECONNREFUSED'];
```

Comment:
Please bring this variable declaration down directly above the if statement, or inline the array. This is purely for readability.

Reply:
Done